processor/elasticinframetricsprocessor/config.go

```diff
@@ -0,0 +1,9 @@
+package elasticinframetricsprocessor // import "github.com/elastic/opentelemetry-collector-components/processor/elasticinframetricsprocessor"
+
+type Config struct {
```

Comment:
Wondering if we should add config tests like those that exists in the processors/components of the upstream repo: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/k8sattributesprocessor/testdata/config.yaml

Reply:
+1 Example test file that uses the testdata dir: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/k8sattributesprocessor/config_test.go#L20

Reply:
I have added the tests.